usaspending_api/awards/v2/filters/filter_helpers.py

```diff
@@ -52,19 +53,35 @@ def date_list_to_queryset(date_list, table):
     for v in date_list:
         # Modified May 2018 so that there will always be a start and end value from combine_date_range_queryset()
```

Comment:
```suggestion
        
```
Comment is out of date now

Reply:
How is the comment out of date?

Reply:
The code's been pretty significantly changed below, and it was modified after May 2018.

Reply:
While that is true, the function `combine_date_range_queryset()` was modified May 2018 to always provide both a start and end date even if one of the fields was not provided. That hasn't changed in this PR.